python/taichi/lang/ops.py

```diff
@@ -129,6 +129,13 @@ def cast(obj, dtype):
     return expr.Expr(_ti_core.value_cast(expr.Expr(obj).ptr, dtype))
 
 
+def group_cast(group, dtype):
```

Comment:
Where is it used?

Reply:
This function is used in my preliminary version, now it is redundant. I will remove it soon.